tools/topology/topology1/CMakeLists.txt

```diff
@@ -142,6 +142,10 @@ set(TPLGS
 	"sof-tgl-max98373-rt5682\;sof-tgl-max98373-rt5682-xperi\;-DAMP_SSP=1\;-DINCLUDE_IIR_EQ=1"
 	"sof-tgl-max98373-rt5682\;sof-tgl-rt5682-ssp0-max98373-ssp2\;-DAMP_SSP=2"
 	"sof-tgl-max98373-rt5682\;sof-tgl-rt5682-ssp0-max98373-ssp2-xperi\;-DAMP_SSP=2\;-DINCLUDE_IIR_EQ=1"
+	"sof-adl-nau8825\;sof-adl-nau8825\;-DBT_OFFLOAD"
+	"sof-adl-nau8825\;sof-adl-max98373-nau8825\;-DCODEC=MAX98373\;-DAMP_SSP=1\;-DBT_OFFLOAD"
+	"sof-adl-nau8825\;sof-adl-max98360a-nau8825\;-DAMP_SSP=1\;-DDIGITAL_AMP\;-DBT_OFFLOAD"
+	"sof-adl-nau8825\;sof-adl-rt1019-nau8825\;-DAMP_SSP=2\;-DDIGITAL_AMP"
```

Comment:
it's much better to have a single file indeed, good change.

I am not clear however on the 'DIGITAL_AMP' definition, it seems it means ifndef (MAX98373)

can you explain what you are trying to do here? which configurations are you trying to isolate?

ifdef(`DIGITAL_AMP',
ifelse(CODEC, `MAX98373',

Reply:
@plbossart , my purpose is separated the 'smart amp' and 'digital amp' for different pipelines configurations.
So ifdef(DIGITAL_AMP) goes to apply MAX98360A and RT1019, 
ifelse(CODEC, MAX98373) goes to apply sof-smart-amplifier.m4.
Perhaps change the name to 'SMART_AMP' for better understanding?

Reply:
Thanks @macchian, yes indeed it'd be better to use an existing term (SMART_AMP) than introduce DIGITAL_AMP which isn't  in use today.